.travis.yml

```diff
@@ -1,75 +1,56 @@
 sudo: required
 language: generic
 services:
-  - docker
-addons:
-  apt:
-    update: true
+    - docker
```

Comment:
I guess the tab changed?

Reply:
yeah, I'll move this all to two spaces in a future patch.

Reply:
actually, it really wants to be four, which is why that moved.